### DISCUSSION TITLE
Change VCPKG_FIXUP_ELF_RPATH inside a port

### DISCUSSION THREAD

**Discussion by asfernandes** [microsoft/vcpkg#35684] [General]
I'm writing a port and testing with linux-x64-dynamic. VCPKG_FIXUP_ELF_RPATH is causing a problem with this package, that already defines `$ORIGIN`-based rpath and is relocatable.

Is it safe to `set(VCPKG_FIXUP_ELF_RPATH OFF)` inside a `portfile.cmake` without it interfere with other ports?

**Comment by Osyotr** [comment]
Yes, portfile is included after triplet.

**Reply by asfernandes** [reply]
If `VCPKG_FIXUP_ELF_RPATH` is set in one port, is it "restored" to not interfere in another?

I tried to manually restore (to ON) it in the end of my port, but then it's likely it was not set to ON in the start.

**Reply by Osyotr** [reply]
`ports.cmake` is executed for each port separately. https://github.com/microsoft/vcpkg/blob/eb33d2f7583405fca184bcdf7fdd5828ec88ac05/scripts/ports.cmake#L153-L173
